runtime/builtin_types.go

```diff
@@ -559,6 +560,35 @@ func builtinRange(f *Frame, args Args, kwargs KWArgs) (*Object, *BaseException)
 	return ListType.Call(f, []*Object{r}, nil)
 }
 
+func builtinRawInput(f *Frame, args Args, kwargs KWArgs) (*Object, *BaseException) {
+	if len(args) > 1 {
+		msg := fmt.Sprintf("[raw_]input expcted at most 1 arguments, got %d", len(args))
+		return nil, f.RaiseType(TypeErrorType, msg)
+	}
+
+	osStdout := os.Stdout
```

Comment:
Use the new grumpy.Stdout variable for output. This will require modifying pyPrint(), Print(), etc. to take a *grumpy.File instead of *os.File. Might be better to add Stdin/out/err and change the print behavior in a separate PR and once that's merged, continue with this one.